api/verify.go

```diff
@@ -87,6 +87,12 @@ func (a *API) Verify(w http.ResponseWriter, r *http.Request) error {
 			user, terr = a.recoverVerify(ctx, tx, params)
 		case emailChangeVerification:
 			user, terr = a.emailChangeVerify(ctx, tx, params)
+			if user == nil && terr == nil {
+				// when double confirmation is required
+				rurl := a.prepRedirectURL("Confirmation link accepted. Please proceed to confirm link sent to the other email", params.RedirectTo)
```

Comment:
@thorwebdev If double confirmation is required and the user confirms one email only, then they will be redirected to either the site url or a redirect url specified in the allowed list with this message. Let me know if this is alright in terms of DX, or if there is a way to make it better.

Reply:
Yes, I think that's fine for now 👍  We can probably improve it once we have a hosted flow.